File: Data Structures & Algorithms/longest-repeating-substring-with-replacement/submission-0.cpp
class Solution {
public:
    int characterReplacement(string s, int k) {
        int l=0,maxf=0,maxL=0;
        unordered_map<char,int> count;
        for(int r=0;r<s.size();r++){
            count[s[r]]++;
            maxf=max(maxf,count[s[r]]);
            while((r-l+1) - maxf > k){
                count[s[l]]--;
                l++;
            }
            maxL=max(r-l+1, maxL );
        }
        return maxL;
    }
};

File: Data Structures & Algorithms/find-duplicate-integer/submission-0.java
class Solution {
    public int findDuplicate(int[] nums) {
        int [] count = new int[nums.length];

        for(int i =0;i<nums.length;i++){
            count[nums[i]]++;
        }
        
        for(int i=0;i<count.length;i++){
            if(count[i]>1){
                return i;
            }
        }
        return -1;
    }
}
